### ISSUE TITLE
pageNavigation metadata in OPF 

### ISSUE THREAD

**Issue by LaZay** [daisy/ace#364]
`<meta property="schema:accessibilityFeature">pageNavigation</meta>`
When adding this accessibility feature in EPUB OPF, Ace is reporting a Violation:
```
'schema:accessibilityFeature' metadata must be set to one of the expected values
Value 'pageNavigation' is invalid for 'schema:accessibilityFeature' metadata
Use one of the metadata values defined by schema.org
```

This metadata seems to exist though, according Daisy web site.
http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityFeature.html#value

**If Daisy is right, would it be possible to debug this please? The violation part of Ace is quite sensitive...**

Nota:
Schema.org web site does not provide the comprehensive list of accessibility values any more. They have been replace by some examples...
https://schema.org/accessibilityFeature
And by a link to a wikipedia page... that is no longer maintained!
https://www.w3.org/wiki/WebSchemas/Accessibility
What is the problem with schema.org "standard"?

**Comment by clapierre** [comment]
Hi @LaZay if you follow the [link in the DAISY KB for "pageNavigation"](http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityFeature.html#pageNavigation) you will notice that this is a proposed value not yet accepted. Perhaps @mattgarrish could add (Proposed) beside those terms not yet accepted on the list of [accessibilityFeature values](http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityFeature.html#value)

**Comment by mattgarrish** [comment]
The error message is too severe. There are preferred values for the accessibility metadata, but there are no invalid values. We have to allow for extensibility (think of all that slash extension stuff that I bet would also be flagged).

It's probably still helpful to flag unexpected values, but it should probably be done at a minor level, as in to check that the value is not a typo.

**Comment by clapierre** [comment]
I agree that ACE should report unexpected values incase there are typos as we have seen this a lot, it should probably also report duplicate entries as this is also something we see a lot which may had different values as would be the case with the accessibilitySummary repeated in the OPF file by mistake.

However @mattgarrish I still feel that the DAISY KB should identify any pending values since looking at the table would imply all values are valid yet digging into them you may discover that some are not yet approved.  BTW Matt, I am referring to the ["Quick Links List"](http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityFeature.html#value)

**Comment by LaZay** [comment]
My fault I did not get down into the [definition page.](http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityFeature.html#pageNavigation) 
I will check this next time, before reporting any strange Violation on github.
No more blind trust of Daisy [list of schema:accessibilityFeature values ](http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityFeature.html#value) in the future... I swear! :-)
We can close the ticket if you want.
